rrfs-test/validated_yamls/templates/obtype_config/atms_n21.yaml

```diff
@@ -0,0 +1,585 @@
+     - obs space:
+         name: atms_n21
+         distribution:
```

Comment:
@keenaneure Thanks a lot for adding atms_n21!

Could you add the common anchor variables to be consistent with other satellite YAML files?

Here is an instruction about the common anchor variables:
https://github.com/guoqing-noaa/satinfo2yaml/blob/main/README.md

Thanks!

Reply:
FYI, this PR https://github.com/NOAA-EMC/RDASApp/pull/357 can also be referenced. Thanks!

Reply:
@keenaneure  Thanks a lot for adding this!